client/src/components/statusInfo.js

```diff
@@ -13,12 +12,24 @@ const StatusInfo = props => {
       {showThumbnails ? (
         <div className="thumbnail-box">
           {Object.keys(props.uploadPercentage).map(key => {
-            return props.uploadPercentage[key] !== 100 ? (
-              <div key={`progress-${key}`}>
-                <p>Connected {key}</p>
+            if (props.uploadPercentage[key].dzsave === -1) {
+              return null;
+            }
+            return props.uploadPercentage[key].dzsave !== 100 ? (
+              <div key={`dzsave-progress-${key}`}>
+                <p>DZSave in progress... {key}</p>
```

Comment:
lets give some styling to this. Like a badge
https://tailwindui.com/components/application-ui/elements/badges